inc/3rd-party/nextgen-gallery/inc/classes/class-imagify-ngg-db.php

```diff
@@ -29,9 +29,9 @@ class Imagify_NGG_DB extends Imagify_Abstract_DB {
 	/**
 	 * Get things started.
 	 *
-	 * @access  public
-	 * @since   1.5
-	 * @author  Jonathan Buttigieg
+	 * @since  1.5
+	 * @access protected
+	 * @author Jonathan Buttigieg
 	 */
 	protected function __construct() {
 		global $wpdb;
```

Comment:
can remove `$wpdb` since it's not used here anymore

Reply:
It is used 3 times :|
But not in `create_table()` indeed.

Reply:
comment didn't go where I wanted to, it was for `create_table()` specifically